src/server/kiwix-serve.cpp

```diff
@@ -123,9 +128,27 @@ void setup_sighandlers()
 {
     set_signal_handler(SIGTERM, &handle_sigterm);
     set_signal_handler(SIGINT,  &handle_sigterm);
+    set_signal_handler(SIGHUP,  &handle_sighup);
 }
 #endif
 
+bool reloadLibrary(kiwix::Manager& mgr, const std::vector<std::string>& paths)
+{
+    try {
+      std::cerr << "Loading the library from the following files:\n";
```

Comment:
Do we want to print this on `cerr` ?
I would say that it should go to `cout`.
But I don't know if there are some special rules for a log coming from a daemon reloading the config.

Reply:
`stderr` is used for diagnostic output if the program sends useful output to `stdout`. `kiwix-serve` doesn't produce any such output so we can print this message to `cout` instead. I have absolutely no preference here.

Reply:
Please move everything to `cout` then.

Reply:
Done (but the output printed in case of an error still goes to `stderr`).